src/domains/apiconfig-app/00_alert_apiconfig_cache.tf

```diff
@@ -0,0 +1,59 @@
+######################
+## API-CONFIG CACHE ##
+######################
+
+locals {
+
+  oracle_version     = "oracle"
+  postgresql_version = "postgresql"
+}
+
+# Pod OutOfMemory - ORACLE
```

Comment:
can you avoid c&p alert code using [for_each](https://github.com/pagopa/pagopa-infra/blob/6c1e5b4d95c2952a3e4d55208e5f9d7251f27379/src/domains/nodo-app/00_alert.tf#L163) ?

Reply:
done